### ISSUE TITLE
Non-working doi

### ISSUE THREAD

**Issue by dli7319** [ruofeidu/DuBibtex#7]
Not sure if this is an real issue or not.
The doi [https://doi.org/10.5555/3045118.3045358](https://doi.org/10.5555/3045118.3045358) doesn't actually work
I think all the 10.5555 are not real DOIs, just some doi-looking ID assigned by ACM

Input:
```
@InProceedings{pmlr-v37-sohl-dickstein15,
  title = 	 {Deep Unsupervised Learning using Nonequilibrium Thermodynamics},
  author = 	 {Sohl-Dickstein, Jascha and Weiss, Eric and Maheswaranathan, Niru and Ganguli, Surya},
  booktitle = 	 {Proceedings of the 32nd International Conference on Machine Learning},
  pages = 	 {2256--2265},
  year = 	 {2015},
  editor = 	 {Bach, Francis and Blei, David},
  volume = 	 {37},
  series = 	 {Proceedings of Machine Learning Research},
  address = 	 {Lille, France},
  month = 	 {07--09 Jul},
  publisher =    {PMLR},
  pdf = 	 {http://proceedings.mlr.press/v37/sohl-dickstein15.pdf},
  url = 	 {https://proceedings.mlr.press/v37/sohl-dickstein15.html},
}
```

Output:
```
@inproceedings{pmlr-v37-sohl-dickstein15,
  title={{Deep Unsupervised Learning Using Nonequilibrium Thermodynamics}},
  author={Sohl-Dickstein, Jascha and Weiss, Eric and Maheswaranathan, Niru and Ganguli, Surya},
  booktitle={Proceedings of the 32nd International Conference on Machine Learning},
  pages={2256--2265},
  year={2015},
  editor={Bach, Francis and Blei, David},
  volume={37},
  series={Proceedings of Machine Learning Research},
  address={Lille, France},
  month={07--09 Jul},
  publisher={PMLR},
  pdf={http://proceedings.mlr.press/v37/sohl-dickstein15.pdf},
  url={https://proceedings.mlr.press/v37/sohl-dickstein15.html},
  doi={10.5555/3045118.3045358}
}
```

**Comment by dli7319** [comment]
5555 is a test prefix for fake dois

https://www.crossref.org/blog/doi-like-strings-and-fake-dois/

**Comment by dli7319** [comment]
Fixed in commit cb0339cc4ded79917848382f9efaaadbdc741618
